tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -635,6 +636,13 @@ include $(MICRO_LITE_BENCHMARKS)
 # Load custom kernel tests.
 include $(MAKEFILE_DIR)/additional_tests.inc
 
+# Generate cc files and headers for all models and bitmaps in project Makefiles.
+# Add all generated cc sources to the build.
+MICROLITE_CC_SRCS += $(shell python3 $(MAKEFILE_DIR)/make_generate_cc_arrays.py $(GENERATED_SRCS_DIR) $(GENERATOR_INPUTS))
```

Comment:
I think we want to avoid putting the generated files into MICROLITE_CC_SRCS.

I'd like to avoid having these be part of libtensorflow-microlite.a but rather as a separate libtflm-data.a or only be compiled with the individual binaries.

Reply:
Done. Moved to microlite_test.